### PR TITLE
updating pkg dependencies

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
**Are you a current Fivetran customer?** 

Fivetran employee 💃 

**What change(s) does this PR introduce?** 

## Updates
- Updated package dependencies for Linkedin Ads v0.7.0, for more information please refer to Linkedin Ads [PR #28](https://github.com/fivetran/dbt_linkedin/pull/28)
- Updated `README` package dependencies to reflect current package versions

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-sheringuyen** [comment]
Note, PR will currently fail as Linkedin PRs have not been merged & released

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# dbt_ad_reporting v1.2.1
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:1]: Since we are updating the LinkedIn Ads package to use the latest breaking change, should this also be a breaking change?

**Review by fivetran-joemarkiewicz — COMMENTED** [review]
Just one comment before approving. I am not 100% set on this being a breaking change, but just think we should think through if an upstream dependency has had a breaking change, should we reflect the same in the downstream (ad reporting) package?

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Pull request merged**